src/store/Store.js

```diff
@@ -67,7 +67,17 @@ class Store {
       chrome.runtime.sendMessage({
         type: DISPATCH_TYPE,
         payload: data
-      }, ({error, value}) => {
+      }, (resp) => {
+        if (resp === undefined) {
+          throw(
+            "Looks like there is an error in the background page. " +
+            "You might want to inspect your background page for more details."
+          );
+        }
+
+        let error = resp.error;
+        let value = resp.value;
```

Comment:
I think we can simplify and destructure by turning this into one line:

`const {error, value} = resp;`